src/main/java/com/spring/board/springboard/post/domain/dto/RequestPostDTO.java

```diff
@@ -0,0 +1,48 @@
+package com.spring.board.springboard.post.domain.dto;
+
+import com.spring.board.springboard.post.domain.Post;
+import com.spring.board.springboard.user.domain.Member;
+
+import java.time.LocalDateTime;
+
+public class RequestPostDTO {
+    private String title;
+    private String content;
+    private Integer memberId;
+
+    public RequestPostDTO() {
+    }
```

Comment:
사용하지 않는다면 private으로 하는 것이 좋겠네요
```suggestion
    private RequestPostDTO() {
    }
```